changelogs/fragments/613-snmp_facts-EndOfMibView.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - snmp_facts module - skip ``EndOfMibView`` values (https://github.com/ansible/ansible/issues/49044).
```

```suggestion
  - snmp_facts - skip ``EndOfMibView`` values (https://github.com/ansible/ansible/issues/49044).
```
there is a small mismatch between changelog examples and the doc statement
```When writing a changelog entry that pertains to a particular module, start the entry with - [module name] - and the following sentence with a lowercase letter.```
i made a PR to fix https://github.com/ansible/ansible/pull/70551